src/main/java/com/cmsc495/ticketsystem/model/MyUser.java

```diff
@@ -0,0 +1,59 @@
+/*MyUser class represents a user and is used define and create myUser objects */
+
+package com.cmsc495.ticketsystem.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class MyUser {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO) //id automatically generated for use as key in database
+    private Long id;
+    private String username;
+    private String password;
+    private String role;
+
+    public MyUser() {};
+
+    public MyUser (String username, String password) {
+        this.username = username;
+        this.password = password;
+        this.role = "User";
```

Comment:
It's fine to have a default here, but on like 17 we should instead have

private String role = "User" 

This makes it so that the default is handled within the class and we can modify it at any time with other added constructors. Just a nitpick.

Reply:
done